### DISCUSSION TITLE
generating bedgraph files from scATAC-seq signal in each cluster

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1439] [General]
Hi Ryan, 

shall we have a scATAC-seq dataset where we call 4 clusters, is there a way to generate bedgraph for each cluster ? Thanks, 

~ Bogdan

**Comment by rcorces** [comment]
https://www.archrproject.com/reference/getGroupBW.html
Please search function documentation before posting. Many functions exist that are not yet highlighted in the manual.
